package.json

```
@@ -5,23 +5,26 @@
   "main": "dist/bundle.js",
   "dependencies": {
     "break_infinity.js": "github:Patashu/break_infinity.js",
-    "esbuild": "^0.11.23",
     "eventemitter3": "^4.0.7",
-    "lz-string": "^1.4.4",
-    "pad-end": "^1.0.2"
+    "lz-string": "^1.4.4"
   },
   "devDependencies": {
     "@types/lz-string": "^1.3.34",
     "@typescript-eslint/eslint-plugin": "^4.22.0",
     "@typescript-eslint/parser": "^4.22.0",
+    "esbuild": "^0.12.9",
     "eslint": "^7.24.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^22.0.0",
     "typescript": "^4.2.4"
   },
   "scripts": {
     "lint": "npx eslint --fix . --ext .ts",
     "csslint": "stylelint Synergism.css --fix",
     "build:esbuild": "npx esbuild src/Synergism.ts --bundle --minify --sourcemap --target=\"chrome58,firefox57,safari11,edge29\" --outfile=\"./dist/out.js\"",
-    "watch:esbuild": "npx esbuild src/Synergism.ts --bundle --minify --sourcemap --target=\"chrome58,firefox57,safari11,edge29\" --outfile=\"./dist/out.js\" --watch"
+    "watch:esbuild": "npx esbuild src/Synergism.ts --bundle --minify --sourcemap --target=\"chrome58,firefox57,safari11,edge29\" --outfile=\"./dist/out.js\" --watch",
+    "check:tsc": "tsc",
```

```suggestion
    "check:tsc": "npx tsc",
```

forced npm to use locally installed typescript